### DISCUSSION TITLE
Feature: Switch组件可以做成运球动作吗

### DISCUSSION THREAD

**Discussion by m1m1sha** [ikun-svelte/ikun-ui#196] [Ideas]
类似 https://github.com/ikun-svelte/ikun-ui/issues/7#issuecomment-1667251573 的svg中运球动作
😁

**Comment by so1ve** [comment]
组件库是正经组件库，可以搞成彩蛋

**Comment by itsoursduty** [comment]
> 组件库是正经组件库，可以搞成彩蛋

做正经的有市场吗。。不如直接做娱乐效果

**Comment by so1ve** [comment]
svelte现在一个成熟组件库都没有你说有没有市场？人家做出来本来就是要在生产里头用的

**Comment by toplyb** [comment]
感觉可以有，开发者可以自己选择是否使用 😂 （纯好玩）

**Comment by NWYLZW** [comment]
加个 type 不就好了，问题不大

**Comment by so1ve** [comment]
可以，加一个选择切换
